### PR TITLE
bugfix: remove No Tickets Available copy

### PR DESCRIPTION
**Pull request by zach-aries**

### CODE REVIEW

File: plugin/css/showpass-flex-box.css [hunk 1]
```diff
@@ -155,7 +155,7 @@
 
 .showpass-flex-table .showpass-layout-flex .showpass-flex-column, .showpass-flex-box .showpass-layout-flex .showpass-flex-column {
     display: flex;
-    align-items: flex-start;
+    align-items: stretch;
     flex-grow: 0 !important;
```
Comment (zach-aries) [inline plugin/css/showpass-flex-box.css]: This makes all card heights equal on grid view

File: plugin/inc/default-list.php [hunk 2]
```diff
@@ -15,36 +15,33 @@
...
+              <a class="showpass-image-banner showpass-hide-mobile <?php if (!$event['external_link']) echo 'open-ticket-widget' ?>" <?php if ( isset($event_data['tracking_id'] ) ) {?> data-tracking="<?php echo $event_data['tracking_id']; ?>" <?php } ?> <?php if ($event['external_link']) { ?>href="<?php echo $event['external_link']; ?>"<?php } else { ?>id="<?php echo $event['slug']; ?>"<?php } ?> style="background-image: url('<?php if ($event['image']) { echo $event['image']; } else { echo plugin_dir_url(__FILE__).'../images/default-square.jpg';}?>');" href="/<?php if ($detail_page) { echo $detail_page; } else { echo 'event-detail'; } ?>/?slug=<?php echo $event['slug']; ?>"></a>
```
Comment (zach-aries) [inline plugin/inc/default-list.php]: was getting a "tracking_id is undefined error" so wrapped in `isset`

**Comment by marc-thomas** [comment]
Hey @zach-aries, there are some merge conflicts here with master.

Added `<?php if ($event['show_eyereturn']) {?> data-eyereturn="<?php echo $event['show_eyereturn']; ?>" <?php } ?>` on master into the button links.

Should all of these conditionals have `if(isset())` ? instead of just `if($event)` ?

**Comment by zach-aries** [comment]
> Hey @zach-aries, there are some merge conflicts here with master.
> 
> Added `<?php if ($event['show_eyereturn']) {?> data-eyereturn="<?php echo $event['show_eyereturn']; ?>" <?php } ?>` on master into the button links.
> 
> Should all of these conditionals have `if(isset())` ? instead of just `if($event)` ?

Okay, I'll handle the merge conflicts.
For the most part, the only case where you would want to do `if($event['key'])` is if you have something like: [ key => '' ] and you want the if to return `true` in that case. If `$event['key'] === ''` `isset` will return false.

**Comment by marc-thomas** [comment]
hey @zach-aries looks like there is a merge conflict here - but we should merge in 

https://github.com/showpass/showpass-wordpress-plugin/pull/17

and 

https://github.com/showpass/showpass-wordpress-plugin/pull/16

First.

**Pull request closed without merging**